featuretools/tests/entityset_tests/test_es.py

```diff
@@ -967,17 +967,30 @@ def test_normalize_entity_new_time_index_additional_success_check(es):
                         copy_variables=[])
 
 
-def test_normalize_time_index_from_none(es):
-    es['customers'].time_index = None
-    es.normalize_entity('customers', 'birthdays', 'age',
-                        make_time_index='date_of_birth',
-                        copy_variables=['date_of_birth'])
-    assert es['birthdays'].time_index == 'date_of_birth'
-    df = es['birthdays'].df
+def test_normalize_time_index_from_none():
+    df = pd.DataFrame({
+        "id": [0, 1, 2, 3],
+        "A": [5, 4, 2, 3],
+        'time': [datetime(2020, 6, 3), (datetime(2020, 3, 12)), datetime(2020, 5, 1), datetime(2020, 4, 22)]
+    })
+    es = ft.EntitySet("es")
+    es = es.entity_from_dataframe(
+        entity_id="data",
+        dataframe=df,
+        index="id")
```

Comment:
the old version of this test is testing both a dask entityset and a pandas one.  We could make a parametrized entityset fixture similar to `es` with this portion of the code, and keep the rest of the code to test the entityset

Reply:
Added in the fixtures so we now have `params=['dd_normalize_es', 'pd_normalize_es']` for this test-- let me know if it looks ok